scripts/update_version.py

```
@@ -0,0 +1,582 @@
+
+This script helps update version numbers across the Honcho repository.
+It handles the main API, Python SDK, and TypeScript SDK in a single operation.
+"""
+
+import os
+import sys
+import subprocess
+import tempfile
+import re
+from datetime import datetime
+from typing import Optional, Dict, List, Tuple
```

Ruff (0.11.9)

`typing.Dict` is deprecated, use `dict` instead

(UP035)